Add message on diff-aware (#28085)
DataDog/documentation: content/en/security/code_security/troubleshooting/_index.md

You can also run datadog-static-analyzer with the `--debug` option to get more information.

> Note that diff-aware works only on feature branches. You can learn more about the [implementation details of diff-aware][13] on the datadog-static-analyzer repository.

Reviewer: The note format matches other usage on this page, and helps it stand out more. Probably not necessary to mention the repo name since the link takes the reader there.
```suggestion
**Note**: Diff-aware works only on feature branches. For more information, learn about the [implementation details of diff-aware][13].
```
